lambdas/data-migration2/main.tf

```
@@ -0,0 +1,106 @@
+locals {
+  lambda_path = "${path.module}/dist/webpack/lambda.zip"
+}
+
+data "aws_iam_policy_document" "lambda_assume_role_policy" {
+  statement {
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+resource "aws_iam_role" "data_migration2" {
+  name                 = "${var.prefix}-data-migration2"
+  assume_role_policy   = data.aws_iam_policy_document.lambda_assume_role_policy.json
+  permissions_boundary = var.permissions_boundary_arn
+
+  tags = var.tags
+}
+
+data "aws_iam_policy_document" "data_migration2" {
+  statement {
+    actions = [
+      "ec2:CreateNetworkInterface",
+      "ec2:DeleteNetworkInterface",
+      "ec2:DescribeNetworkInterfaces",
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:DescribeLogStreams",
+      "logs:PutLogEvents"
+    ]
+    resources = ["*"]
+  }
+
+  statement {
+    actions = [
+      "dynamodb:Scan",
```

I needed to make the following update for the script to run as a lambda in the clear (e.g. not on ECS): 

```suggestion
      "dynamodb:Scan",
      "dynamodb:GetItem",
```